xml/ha_storage_protection.xml

```diff
@@ -705,7 +705,15 @@ Timeout (msgwait)  : 40
    </sect3>
    <sect3 xml:id="sec.ha..storageprotection.sgpersist">
     <title>Configuring an sg_persist Resource</title>
-    <para/>
+    <note>
+     <title>SCSI Disk Compatibility</title>
+     <para>
+      Any backing disks for <literal>sg_persist</literal> must be SCSI disk
+      compatible. <literal>sg_persist</literal> only works for devices like SCSI
+      disks or iSCSI LUNs. Do <emphasis>not</emphasis> use it for IDE, SATA, LVM,
```

Comment:
DO remove LVM here to give too much constrains to our document. In fact, the storage stack do support persistent reservation if all backing disks for lvm are SCSI compatible. Sorry for my initial misleading message ;)

Reply:
No problem, removed LVM now.